storage.go

```diff
@@ -6,6 +6,12 @@ import (
 	"path"
 )
 
+const (
+	StorageTypeStandard = "standard"
```

Comment:
I suggest creating a new type. Example code:
`type StorageType string`
`const Standard StorageType = "standard"`
Doing this way we can change the signature of a function like `somefunc(typeStorage StorageType)` so we can strictly limit the options for input `typeStorage`. Also this is my current coding convention.